packages/common/src/types/index.ts

```diff
@@ -71,11 +92,11 @@ export type CurrentUser = {
   avatarUrl: string | undefined;
   jwt: string | undefined;
   subscription:
```

Comment:
@SaraVieira This can just be

```js
subscription?: {
  amount: string;
  since: string;
};
```

Reply:
They need to be there but will return undefined if not set but they need to exist so its not actually optional